### PR TITLE
gtx_extended_exponential

### PR DESCRIPTION
**Pull request by plasmacel**
Issue #391 

Todo: should perform a GLM_HAS_CBRT style check of the availability of C++11 std::cbrt function.

### CODE REVIEW

**Comment by svenstaro** [comment]
You should really squash your commits. The commit history is really ugly and hard to follow.

**Comment by Groovounet** [comment]
Too many randoms stuff. Sorry but I just don't have the time to review such pull request.

Making a pull request on a specific thing, is much more manageable.

Thanks,
Christophe
